Tests/UnitTests/Scripts/tests.ts

```diff
@@ -60,7 +68,7 @@ describe("AbortController", function () {
 });
 
 describe("XMLHTTPRequest", function () {
-    function createRequest(method, url, body, responseType) {
+    function createRequest(method: string, url: string, body: any = undefined, responseType: any = undefined): Promise<XMLHttpRequest> {
```

Comment:
Can you use `unknown` instead of `any`?
```suggestion
    function createRequest(method: string, url: string, body: uknown = undefined, responseType: unknown = undefined): Promise<XMLHttpRequest> {
```